helm/cluster-cloud-director/values.schema.json

```diff
@@ -322,6 +322,55 @@
                             "pattern": "^[a-z0-9-]{1,63}$"
                         }
                     }
+                },
+                "shell": {
+                    "type": "object",
+                    "title": "Shell access",
+                    "properties": {
+                        "osUsers": {
+                            "type": "array",
+                            "title": "OS Users",
+                            "description": "Configuration for OS users in cluster nodes.",
+                            "items": {
+                                "type": "object",
+                                "title": "User",
+                                "required": [
+                                    "name"
+                                ],
+                                "properties": {
+                                    "name": {
+                                        "type": "string",
+                                        "title": "Name",
+                                        "description": "Username of the user.",
+                                        "minLength": 2,
+                                        "pattern": "^[a-z][-a-z0-9]+$"
```

Comment:
This is a new constraint for user names. This was previously not constrained.

Reply:
makes sense to me